fvtest/porttest/omrsockTest.cpp

```diff
@@ -509,6 +515,41 @@ TEST(PortSockTest, two_socket_stream_communication)
 	omrsock_socket_t connectedClientStreamSocket = NULL;
 	EXPECT_EQ(OMRPORTLIB->sock_accept(OMRPORTLIB, serverStreamSocket, &connectedClientStreamSockAddr, &connectedClientStreamSocket), 0);
 
+	char msg[100] = "This is an omrsock test for 2 socket stream communications.";
```

Comment:
I think this would be better as a pointer to the string literal:
```suggestion
	const char *msg = "This is an omrsock test for 2 socket stream communications.";
```